app/video_routes.py

```diff
@@ -0,0 +1,115 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from dotenv import load_dotenv
+load_dotenv()
+
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@videos_bp.route("", methods=["GET"])
+def get_video():
+    videos = Video.query.all()
+
+    response = []
+    for video in videos:
+        response.append(video.to_dict())
+
+    return jsonify(response), 200
+
+@videos_bp.route("", methods=["POST"])
+def create_video():
+    request_body = request.get_json()
+    try:
+        new_video = Video.from_dict(request_body)
+        db.session.add(new_video)
+        db.session.commit()
+
+        return jsonify(new_video.to_dict()), 201
+
+    except:
+        if "title" not in request_body.keys():
+            response = {
+                "details" : "Request body must include title."
+            }
+        elif "release_date" not in request_body.keys():
+            response = {
+                "details" : "Request body must include release_date."
+            }
+        elif "total_inventory" not in request_body.keys():
+            response = {
+                "details" : "Request body must include total_inventory."
+            }
```

Comment:
keys are the default iterator for dictionaries so we can actually omit the keys() method and this conditional will work the same way.